highlighters/main/main-highlighter.zsh

```diff
@@ -432,6 +449,56 @@ _zsh_highlight_highlighter_main_paint()
       fi
     fi
 
+    # Expand aliases.
+    # TODO: this should be done iteratively, e.g., 'alias x=y y=z z=w\n x' 
+    #       And then the entire 'alias' branch of the 'case' statement should
+    #       be done here.
+    () {
+      # TODO: path expansion should happen _after_ alias expansion
+      _zsh_highlight_main_highlighter_expand_path $arg
+      local expanded_arg="$REPLY"
```

Comment:
Any reason to keep this variable?

Reply:
The argument would be readability through naming the return value, but I've removed it.